src/java/picard/sam/markduplicates/EstimateLibraryComplexity.java

```diff
@@ -127,13 +127,21 @@
             "group size would be approximately 10 reads.")
     public int MAX_GROUP_RATIO = 500;
 
+    @Option(doc = "Barcode SAM tag (ex. BC)", optional=true)
```

Comment:
+1 to Nils's suggestion about promulgating new tags sooner rather than later (though as these are alignment record tags, the plea to email samtools-devel is in §1.5 ¶2).

When someone does this, please also expand on @wheaton5's comment that _the BC field in our bams is the sample_index_.  What is your _sample_index_?  If this is not similar to

> Barcode sequence, with any quality scores stored in the QT tag

which is how the SAM spec describes the **BC:Z** tag, then there may be a problem.